dockerfiles/Dockerfile.ci.os

```diff
@@ -0,0 +1,34 @@
+FROM python:3.12-slim
+
+ENV STAC_FASTAPI_TITLE="stac-fastapi-opensearch"
```

Comment:
Sorry but could you remove the ENV lines here? The environment variables should be set when you run the Docker image. Either from an ,env file or as `-e` tags in the `docker run` command. They don't need to be included in the Dockerfile.